File: php-laraval-app/app/Models/BlockDay.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;

class BlockDay extends Model
{
    use HasFactory;

    /**
     * Indicates if the model should be timestamped.
     *
     * @var bool
     */
    public $timestamps = true;

    protected $fillable = [
        'block_date',
        'start_block_time',
        'end_block_time',
    ];

    protected $casts = [
        'created_at' => 'datetime',
        'updated_at' => 'datetime',
    ];
}

File: dotnet/src/Downstairs.Infrastructure/Persistence/Models/BlockDay.cs
namespace Downstairs.Infrastructure.Persistence.Models;

public partial class BlockDay
{
    public long Id { get; set; }

    public DateOnly BlockDate { get; set; }

    public TimeOnly? StartBlockTime { get; set; }

    public TimeOnly? EndBlockTime { get; set; }

    public DateTime? CreatedAt { get; set; }

    public DateTime? UpdatedAt { get; set; }
}